src/mixins/java/org/spongepowered/common/mixin/optimization/world/level/block/entity/BellBlockEntityMixin_Optimization_BellLeak.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.mixin.optimization.world.level.block.entity;
+
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.level.block.entity.BellBlockEntity;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import org.spongepowered.common.mixin.core.world.level.block.entity.BlockEntityMixin;
+
+import java.util.List;
+
+@Mixin(BellBlockEntity.class)
+public abstract class BellBlockEntityMixin_Optimization_BellLeak extends BlockEntityMixin {
+
+    @Shadow private List<LivingEntity> nearbyEntities;
+    @Shadow private long lastRingTimestamp;
+
+    @Inject(method = "tick", at = @At(value = "HEAD"))
+    private void impl$onStopResonating(final CallbackInfo ci) {
```

Comment:
```suggestion
    private void bellleakfix$clearNearbyEntitiesWhenDoneResonating(final CallbackInfo ci) {
```

so that it's clearer when reviewing the method calls if this ever shows up in a stack trace ro something.